composer/yamls/tpu/resnet50.yaml

```diff
@@ -0,0 +1,40 @@
+callbacks:
```

Comment:
This `yaml` doesn't seem correct -- synthetic data, etc. I also don't think we need a device-specific YAML, right? We should just be able to use the `resnet50.yaml` and change the device to `tpu`. Prefer not to maintain duplicate yamls unless neccessary.

Reply:
happy to remove this one and add it to the private repo.